codegen-renderers/src/test/kotlin/io/vrap/codegen/languages/TestCodeGenerator.kt

```diff
@@ -31,16 +32,18 @@ import javax.tools.ToolProvider
 class TestCodeGenerator {
 
     companion object {
-        private val userProvidedPath = System.getenv("TEST_RAML_FILE")
-        private val apiPath : Path = Paths.get(if (userProvidedPath == null) "../api-spec/api.raml" else userProvidedPath)
-        val apiProvider: ApiProvider = ApiProvider(apiPath)
```

Comment:
I think that our tests are failing because you remove the above 2 lines, please try to revert your change and then hopefully our tests run succesfully again 😄

Reply:
good catch 👍